setup.py

```diff
@@ -66,11 +66,13 @@
 XPALIB_DIR = "cextern/xpa-2.1.15"
 CONF_H_NAME = os.path.join(XPALIB_DIR, "conf.h")
 
+# Output from Cythonizing
+CYTHON_SOURCE = "wrappers/xpa.c"
+
 # check if cython is available.
 try:
     from Cython.Build import cythonize
     cythonize("wrappers/xpa.pyx")
-    CYTHON_SOURCE = "wrappers/xpa.c"
     have_cython=True
 except ImportError:
     print("Unable to load Cython")
```

Comment:
Most of this should actually be blown away entirely.  astropy-helpers already has built-in facilities for importing Cython as-needed, running cythonize, etc.  I can open a separate PR to clean up this setup.py so it just uses astropy-helpers to do all this.  @sosey ?